src/org/omegat/core/data/ProjectProperties.java

```diff
@@ -110,7 +110,11 @@ public ProjectProperties(File projectDir) throws Exception {
         setTargetLanguage("UK-UA");
 
         loadProjectSRX();
-        loadProjectFilters();
+        try {
+            loadProjectFilters();
+        } catch (IOException ignored) {
+            projectFilters = new Filters();
```

Comment:
This makes changing behavior of the error handling of project loading.
In a previous case, project loading become failed when filter configuration `filters.xml` corrupted.
This changes the behavior when the filter configuration is corrupted, it will be reset as default.
Is it intended?

Reply:
I think it natural behavioir.